From eb4af78bb314ecc1600dab571648b3b498a1ac2b Mon Sep 17 00:00:00 2001
From: qwang2 <quanyang.wang@windriver.com>
Date: Tue, 19 Jun 2018 15:25:12 +0800
Subject: [PATCH] axxiaarm64: add compatible property for sysfpga@40 node

Add missing compatible property "sysfpga" for node "sysfpga@40" for
axxiaarm64 board.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm64/boot/dts/intel/axm5616-victoria.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/intel/axm5616-victoria.dts b/arch/arm64/boot/dts/intel/axm5616-victoria.dts
index a9a99e0..f0be05d 100644
--- a/arch/arm64/boot/dts/intel/axm5616-victoria.dts
+++ b/arch/arm64/boot/dts/intel/axm5616-victoria.dts
@@ -196,6 +196,7 @@
 	status = "okay";
 
 	sysfpga@40 {
+		compatible = "sysfpga";
 		reg = <0x40>;
 	};
 };
-- 
1.7.9.5

